ApplySyntax.sublime-settings

```diff
@@ -24,7 +24,18 @@
 
     "default_syntaxes": [
         {
-            // I put XML first because of files like *.tmLanguage. It is unlikely
+            // The MXML rules come before the generic XML rules because they
+            // need syntax highlighting for the CDATA sections, but they do not
+            // have any characteristics that distinguish themselves from XML
+            // aside from their file extension
+            "syntax": "MXML",
+            "extensions": ["mxml"],
+            "rules": [
+                {"first_line": "^<\\?xml"}
```

Comment:
Probably can remove the rules and just use the extensions.  That way xml syntax will be used for files that don't end in mxml.